### PR TITLE
Milestone 3: Build React-Redux App

### PR DESCRIPTION
**Pull request by BenMKT**
# Project implementations: 

- Developed a React application using Redux for state management.
- Created a homepage with a list of items that can be filtered based on user input.
- Integrated an API of choice to retrieve numeric data for the categories.
- Designed and implemented a details page to display detailed data for a selected category.
- Created an About page with more details about the app.
- Implemented navigation between the homepage and details page.
- Ensured code adheres to HTML, CSS, and JavaScript best practices.
- Added appropriate documentation for the project in a professional manner.
- Ensured there are no linter errors in the codebase.
- Followed the correct Gitflow process for version control.

API: 

- Used the Jikan API.
- Selected the Financial Modeling Prep API for obtaining stock data.
- Integrated the API into the application to retrieve numeric values based on parameters.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  text-transform: capitalize;
+  border: none;
+  outline: none;
+  font-family: 'Lato', sans-serif;
+  list-style: none;
```
Comment (AbelG101) [inline src/App.css]: - [x] Kindly make sure to use all the fonts specified in the original design as I noticed you're not using the "gill sans" font please recheck sections of the app that use this font and apply it accordingly.

File: src/components/Homepage.js
Comment (AbelG101) [inline src/components/Homepage.js:1]: - [x] As part of the project requirements the Home page should have a list of categories that could be filtered using given parameters (e.g. movie rating, in your case) so please make sure to add this.

File: src/tests/categoryReducer.test.js
Comment (AbelG101) [inline src/tests/categoryReducer.test.js:1]: - [ ] Kindly make sure to add unit tests for several components of the app and some pure functions in the Redux code as I noticed you're only testing the category reducer.
- [ ] Also please make sure to add Integration tests to your application you can perform a mocking data fetching and Redux Store connection.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,181 @@
...
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## 🚀 You can check the Live Demo here<a name="live-demo"></a>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+[Anime/Details Live Demo](https://benmkt-react-redux-project.onrender.com) 
 
-### `npm run eject`
+[Project Presentation/Walk-through](https://www.loom.com/share/29568f281a4c4d5ba0b97c5cf48880bb?sid=621b4da3-af5f-4ac2-b93c-40c745ba94cc)
```
Comment (AbelG101) [inline README.md:63]: - [ ] You've done a great job giving an overview of your codebase but you should also explicitly mention a piece of code that you're proud of or find interesting as per the project requirements.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @BenMKT 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- App content has been personalized ✔
- Page navigation works properly ✔
- Correct use of Git Flow ✔
- Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbelG101** [comment]
Hello @BenMKT please refrain from making changes after you request a review as it will alter the review and cause confusion I noticed you added 2 new commits.

File: src/App.css (hunk 1 above)
Comment (BenMKT) [reply src/App.css]: - Changed font family to Gill Sans as per project requirements

File: src/components/Homepage.js
Comment (BenMKT) [reply src/components/Homepage.js:1]: - Added input field to allow the user to filter shows by movie ratings.

File: src/tests/categoryReducer.test.js
Comment (BenMKT) [reply src/tests/categoryReducer.test.js:1]: - Added tests for `Detailspage` page and `NavBarpage` page.

**Review by frank1738 — APPROVED** [review]
Hi @BenMKT , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Good work adding an additional filter feature👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**